main/src/main/java/org/mobilitydata/gtfsvalidator/validator/MissingTripEdgeValidator.java

```diff
@@ -36,46 +38,44 @@
 
   @Override
   public void validate(NoticeContainer noticeContainer) {
-    Multimaps.asMap(stopTimeTable.byTripIdMap())
-        .forEach(
-            (tripId, stopTimesForTrip) -> {
-              if (stopTimesForTrip.isEmpty()) {
-                return;
-              }
-              GtfsStopTime tripFirstStop = stopTimesForTrip.get(0);
-              GtfsStopTime tripLastStop = stopTimesForTrip.get(stopTimesForTrip.size() - 1);
-              if (!tripFirstStop.hasArrivalTime()) {
-                noticeContainer.addValidationNotice(
-                    new MissingTripEdgeNotice(
-                        tripFirstStop.csvRowNumber(),
-                        tripFirstStop.stopSequence(),
-                        tripId,
-                        "arrival_time"));
-              }
-              if (!tripFirstStop.hasDepartureTime()) {
-                noticeContainer.addValidationNotice(
-                    new MissingTripEdgeNotice(
-                        tripFirstStop.csvRowNumber(),
-                        tripFirstStop.stopSequence(),
-                        tripId,
-                        "departure_time"));
-              }
-              if (!tripLastStop.hasArrivalTime()) {
-                noticeContainer.addValidationNotice(
-                    new MissingTripEdgeNotice(
-                        tripLastStop.csvRowNumber(),
-                        tripLastStop.stopSequence(),
-                        tripId,
-                        "arrival_time"));
-              }
-              if (!tripLastStop.hasDepartureTime()) {
-                noticeContainer.addValidationNotice(
-                    new MissingTripEdgeNotice(
-                        tripLastStop.csvRowNumber(),
-                        tripLastStop.stopSequence(),
-                        tripId,
-                        "departure_time"));
-              }
-            });
+    for (Entry<String, List<GtfsStopTime>> entry : Multimaps.asMap(stopTimeTable.byTripIdMap())
+        .entrySet()) {
+      String tripId = entry.getKey();
+      List<GtfsStopTime> stopTimesForTrip = entry.getValue();
+      GtfsStopTime tripFirstStop = stopTimesForTrip.get(0);
+      GtfsStopTime tripLastStop = stopTimesForTrip.get(stopTimesForTrip.size() - 1);
+      if (!tripFirstStop.hasArrivalTime()) {
+        noticeContainer.addValidationNotice(
+            new MissingTripEdgeNotice(
+                tripFirstStop.csvRowNumber(),
+                tripFirstStop.stopSequence(),
+                tripId,
+                "arrival_time"));
+      }
+      if (!tripFirstStop.hasDepartureTime()) {
+        noticeContainer.addValidationNotice(
+            new MissingTripEdgeNotice(
+                tripFirstStop.csvRowNumber(),
+                tripFirstStop.stopSequence(),
+                tripId,
+                "departure_time"));
```

Comment:
DEPARTURE_TIME_FIELD_NAME here and below.

You can import both static fields:

import static org.mobilitydata.gtfsvalidator.table.GtfsStopTimeTableLoader.DEPARTURE_TIME_FIELD_NAME;